src/pages/index.tsx

```diff
@@ -28,8 +32,58 @@ const App = () => {
         <Flex h="calc(100vh - 3rem)">
           <Sidebar />
           <EditorErrorBoundary>
-            <Box bg="white" flex={1} position="relative">
-              <Editor />
+            <Box bg="#edf2f6" flex={1} position="relative">
+              <Flex
+                w="100%"
+                p={4}
+                backgroundSize="20px 20px"
+                bgColor="#edf2f6"
+                align="center"
+                justify="center"
+                position="absolute"
+                zIndex={10}
+                boxShadow="lg"
+              >
+                <IconButton
+                  icon={<HiOutlineDesktopComputer />}
+                  size="lg"
+                  fontSize="30px"
+                  color={editorWidth === '100%' ? 'teal.500' : 'black.500'}
+                  aria-label="Desktop version"
+                  onClick={() => setEditorWidth('100%')}
+                />
+
+                <IconButton
+                  icon={<MdOutlineTabletMac />}
+                  size="lg"
+                  color={editorWidth === '48em' ? 'teal.500' : 'black.500'}
```

Comment:
you can use the breakpoint values from the Chakra UI theme object 

```
const theme = useTheme()
theme.breakpoint
```

https://chakra-ui.com/docs/styled-system/theme#breakpoints